common/djangoapps/student/cookies.py

```diff
@@ -94,38 +139,25 @@ def set_logged_in_cookies(request, response, user):
         pass
 
     # Convert relative URL paths to absolute URIs
-    for url_name, url_path in header_urls.iteritems():
+    for url_name, url_path in six.iteritems(header_urls):
         header_urls[url_name] = request.build_absolute_uri(url_path)
 
+    enrollments = []
+    for enrollment in CourseEnrollment.enrollments_for_user(user):
+        enrollments.append({
+            'course_run_id': six.text_type(enrollment.course_id),
+            'seat_type': enrollment.mode
+        })
+
     user_info = {
         'version': settings.EDXMKTG_USER_INFO_COOKIE_VERSION,
         'username': user.username,
         'email': user.email,
```

Comment:
@clintonb we don't appear to use this anywhere. Can it be removed, either as part of this PR or in a separate commit/PR?